README.md

```diff
@@ -1,34 +1,56 @@
+![Screenshot of value suggestions for a Liberty variable in a server.env file. If there is a default value, it is preselected](./docs/images/variable-value-completion.png "Completion suggestions for Liberty variable values in server.env")
+* Completion for Liberty config in `server.xml`
 
-1. Create a VS Code workspace with both [open-liberty-tools-vscode](https://github.com/OpenLiberty/open-liberty-tools-vscode) and this project at the root of the workspace. The folder structure should look something like this:
-```
-| > open-liberty-tools-vscode
-| v liberty-language-server
-| | > lemminx-liberty
-| | > liberty-ls
-```
+![Screenshot of Liberty feature suggestions in a feature block in a server.xml file](./docs/images/feature-completion.png "Completion suggestions for Liberty configuration in server.xml")
 
-2. In the `open-liberty-tools-vscode` directory, run `npm run build`.
+### Hover on Liberty server configuration files
+* Hover for Liberty properties and variables
 
-3. Open the debug view, select and launch `Run Extension (open-liberty-tools-vscode)`. It will open a new window with the extension running in debug mode.
+![Screenshot of a documentation dialog appearing when hovering over a Liberty property in a bootstrap.properties file](./docs/images/property-hover.png "Hover on Liberty properties in bootstrap.properties")
+![Screenshot of a documentation dialog appearing when hovering over a Liberty variable in a server.env file](./docs/images/variable-hover.png "Hover on Liberty server variables in server.env")
+* Hover for Liberty config in `server.xml` 
 
-4. In the same debug view, now select and launch one of the following to debug each respective project:
-    * `Debug attach liberty-lemminx`
-    * `Debug attach liberty-ls` (in progress)
\ No newline at end of file
+![Screenshot of feature documentation appearing when hovering over a Liberty feature in a server.xml file](./docs/images/feature-hover.png "Hover on Liberty features in server.xml")
+
+### Diagnostics on Liberty server configuration files
+* Diagnostics on Liberty properties and variables
+
+![Screenshot showing diagnostics marking an invalid value for a Liberty property in a bootstrap.properties file. Hovering over the diagnostic will provide more details.](./docs/images/property-diagnostic.png "Diagnostics on Liberty properties in bootstrap.properties")
+![Screenshot showing diagnostics marking an invalid value for a Liberty variable in a server.env file. Hovering over the diagnostic will provide more details.](./docs/images/variable-diagnostic.png "Diagnostics on Liberty variables in server.env")
+* Diagnostics for Liberty features
+
+![Screenshot showing diagnostics marking an invalid feature defined in a server.xml file. Hovering over the diagnostic will provide more details.](./docs/images/feature-diagnostic.png "Diagnostics on Liberty features in server.xml")
```

Comment:
Should we add a screenshot for diagnostics on server.xml elements that are not Liberty features?

Reply:
I think that would be a good idea. I'm not familiar with the LemMinX side; what other diagnostic examples should we showcase?

Reply:
I don't think it is necessary. We've covered that diagnostics are provided for server.xml, which is the point.